PrecipTable.ipynb

```diff
@@ -64,7 +64,6 @@
    },
    "outputs": [],
    "source": [
-
```

Comment:
All differences are a result of updates to the notebook's metadata caused by executing the notebook. No changes made to the notebook itself.